Sources/SwiftSyntax/generated/SyntaxKind.swift

```diff
@@ -16,8 +16,6 @@
 @frozen // FIXME: Not actually stable, works around a miscompile
 public enum SyntaxKind {
   case token
-  case accessPathComponent
-  case accessPath
```

Comment:
Same here, just a note to myself, no action required on your side: This is not source compatible if we are renaming a syntax node’s type name. We might be able to mitigate this by defining a static member on `SyntaxKind` as follows

```swift
public extension SyntaxKind {
  @available(*, deprecated, renamed: "importPath")
  var accessPath: SyntaxKind { .importPath }
}
```

This way we will only break clients that are exhaustively switching over `SyntaxKind`, which is fairly unlikely since the enum is so huge.